bpf/kmesh/skb_verdict.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <linux/bpf.h>
+#include <bpf/bpf_helpers.h>
+
+static long (*bpf_migration_socket)(struct __sk_buff *skb) = (void *)163;
```

Comment:
Is this the place for length? Would it be better to use `length` ewplace `long`

Reply:
This code is used for socket migration. This function will not be brought online currently.